### PR TITLE
Update install instructions for local installation

### PR DESCRIPTION
**Pull request by hagenw**
I don't know if it is desired to run the benchmark outside of a docker container, but i think it is a nice addition to also add some instructions to run the benchmark just in a virtual environment.

I also added `aubio==0.4.9` to `requirementst.txt`, but maybe there was a particular reason why you installed it explicitely in the Docker container instead?

### CODE REVIEW

**Comment by faroit** [comment]
> I don't know if it is desired to run the benchmark outside of a docker container, but i think it is a nice addition to also add some instructions to run the benchmark just in a virtual environment.

well, if you can make it. I was trying hard for several days. But neither pip nor conda worked for me on OSX. There where problems with either gstreamer (conda), aubio (pip) or torchaudio (both)...
Feel free to try it, but I really don't know if its worth the efford. Codecs are a mess ;-)

> I also added aubio==0.4.9 to requirementst.txt, but maybe there was a particular reason why you installed it explicitely in the Docker container instead?

yes the pip wheels didn't ship all the codecs, at least not on mac, thats why i installed it from source

File: Dockerfile [hunk 1]
```diff
@@ -49,8 +49,5 @@ WORKDIR /app
 RUN pip install pycairo
 RUN pip install --requirement /app/requirements.txt
 
-# install aubio 0.4.9 from source for ffmpeg
-RUN git clone https://git.aubio.org/aubio/aubio aubio_src && cd aubio_src && git checkout 0.4.9 && ./setup.py clean && pip install -v .
```
Comment (faroit) [inline Dockerfile]: Did you test the docker file when aubio is installed from pypi?

**Comment by hagenw** [comment]
Works perfectly here under Linux
benchmark_np

File: Dockerfile (hunk 1 above)
Comment (hagenw) [reply Dockerfile]: Not yet, as I have no ready docker installation at hand.

Comment (hagenw) [reply Dockerfile]: But I can check tomorrow.

**Comment by faroit** [comment]
sorry for the long delay 👍

**Comment by hagenw** [comment]
Cool, thanks.

**Pull request merged**